website_cookiebot/data/cookies_policy.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="cookies" model="ir.ui.view">
+            <field name="name">Cookies Policy</field>
+            <field name="type">qweb</field>
```

Comment:
Instead of creating a qweb template in this form, you should use the `<template>` helper if I'm not wrong.